src/commands/sites/create.js

```diff
@@ -19,7 +20,20 @@ class SitesCreateCommand extends Command {
 
     let name = flags.name
     if (!name) {
-      console.log('Choose a site name or leave blank for a random name. You can update later.')
+      const userName = await api.getCurrentUser()
+      const suggestions = [
+        `super-cool-site-by-${userName.slug}`,
+        `the-awesome-${userName.slug}-site`,
+        `${userName.slug}-makes-great-sites`,
+        `netlify-thinks-${userName.slug}-is-great`,
+        `the-great-${userName.slug}-site`,
+        `isnt-${userName.slug}-awesome`,
+        `${userName.slug}-blog`
```

Comment:
I love these! We don't know that they're making a blog though so this one should be altered or removed.